src/Core/Pages/Vault/CipherAddEditPage.xaml.cs

```diff
@@ -268,6 +268,15 @@ private async void ScanTotp_Clicked(object sender, System.EventArgs e)
                         {
                             await Navigation.PopModalAsync();
                             await _vm.UpdateTotpKeyAsync(key);
+
+#if IOS
+                            //Workaround: To avoid a bug that incorrectly sets the TextColor when changing text
+                            // programatically we need to set it to null and back to the correct color
+                            // MAUI issue https://github.com/dotnet/maui/pull/20100
+                            _loginTotpEntry.TextColor = null;
+                            var color = ThemeManager.GetResourceColor("TextColor");
+                            _loginTotpEntry.TextColor = color;
+#endif
```

Comment:
🎨 No need for the extra variable, and HACK might be better given that it's tracked by Tasks:
```suggestion
#if IOS
                            // HACK: To avoid a bug that incorrectly sets the TextColor when changing text
                            // programatically we need to set it to null and back to the correct color
                            // MAUI issue https://github.com/dotnet/maui/pull/20100
                            _loginTotpEntry.TextColor = null;
                            _loginTotpEntry.TextColor = ThemeManager.GetResourceColor("TextColor");
#endif
```